tests/unit/conditions/test_condition_validity.py

```diff
@@ -89,12 +89,25 @@ def test_invalid_contract_condition():
     # invalid ABI
     with pytest.raises(ReencryptionCondition.InvalidCondition):
         _ = ContractCondition(
-                contract_address="0xaDD9D957170dF6F33982001E4c22eCCdd5539118",
-                method="getPolicy",
-                chain=TESTERCHAIN_CHAIN_ID,
-                function_abi=["rando ABI"],
-                return_value_test=ReturnValueTest('!=', 0),
-                parameters=[
-                    ':hrac',
-                ]
-            )
+            contract_address="0xaDD9D957170dF6F33982001E4c22eCCdd5539118",
+            method="getPolicy",
+            chain=TESTERCHAIN_CHAIN_ID,
+            function_abi=["rando ABI"],
```

Comment:
(included in https://github.com/theref/nucypher/pull/1)

```suggestion
            function_abi={"rando": "ABI"},
```